validate-schema.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: CC0-1.0
+
+import os
+
+import xmlschema
+
+def getAllXMLFiles(srcDir):
+    # recursively gets all files in srcDir with .xml extension.
+    paths = []
+    for root, ds, fs in os.walk("src"):
+        for f in fs:
+            if f.endswith(".xml"):
+                paths.append(os.path.join(root, f))
+    return paths
+
+if __name__ == "__main__":
+    spdxSchema = xmlschema.XMLSchema("schema/ListedLicense.xsd")
+    paths = getAllXMLFiles("src")
+
+    print(f"Checking {len(paths)} licenses and exceptions for schema validation...")
+    for xf in paths:
+        spdxSchema.validate(xf)
```

Comment:
Alternatively, call `is_valid`, so that you can check all files without raising an exception and stop checking the rest.

If it returns false, add it to a list of "bad" ones. At the end of the loop if the bad list is not empty, print them and raise an exception.

Reply:
Good call -- I'll give that a try, will update...